amazon-bedrock-csv-chatbot-poc/app.py

```diff
@@ -0,0 +1,45 @@
+import streamlit as st
```

Comment:
Check out the front end here: https://github.com/aws-samples/genai-quickstart-pocs/blob/main/amazon-bedrock-summarization-long-document-poc/app.py